layouts/shortcodes/config-file-k8s-istio.html

```diff
@@ -1 +1 @@
-kfctl_k8s_istio.v1.0.2.yaml
\ No newline at end of file
+kfctl_k8s_istio.yaml
```

Comment:
seems manifest file here point to master. https://github.com/kubeflow/manifests/blob/316c3f16698fce5de8349daf1a59756aeaeec18a/kfdef/kfctl_k8s_istio.yaml#L340

Is that expected?

Reply:
Thanks for the pointer, totally missed that.
Let me make a PR to the manifests repo and get back to you